device/pensando/arm64-elba-asic-r0/Pensando-elba/port_config.ini

```diff
@@ -0,0 +1,3 @@
+# name          lanes		alias		speed	autoneg	fec
+Ethernet1	29		Ethernet1	100000	on	rs
```

Comment:
the lanes should be 4 lanes, btw, for smartswitch one isn't the link should be 1x200G port?

Reply:
Corrected the lanes.